spec/lib/miq_expression_spec.rb

```diff
@@ -2891,7 +2893,8 @@
       expect(col_info).to match(
         :data_type                      => :string,
         :excluded_by_preprocess_options => false,
-        :include                        => {},
+        :format_sub_type                => :string,
+        :include                        => {:host => {}},
```

Comment:
according to the tag, this does indeed need the host model.

I stand by this change